### ISSUE TITLE
NTR: Supravalvar pulmonary stenosis

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#7667]
**Preferred term label:**
Supravalvar pulmonary stenosis

**Synonyms (exact)**
Supravalvar pulmonnic stenosis, Supravalvular pulmonary stenosis, Supravalvular pulmonic stenosis

**Definition (free text, please give PubMed ID)**
A cardiovascular malformation associated with narrowing at the level of the pulmonary sinotubular junction above the pulmonic valve.
PMID:34304616

**Parent term (use hpo.jax.org/app)**
Pulmonic stenosis HP:0001642 (per #7664)

**Xrefs**
SNOMEDCT_US:52757001, MONDO:0017870

Your nano-attribution (ORCID)

**Comment by pnrobinson** [comment]
added. thanks!
